PinpointKit/PinpointKitTests/SystemLogCollectorTests.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  SystemLogCollectorTests.swift
+//  PinpointKit
+//
+//  Created by Andrew Harrison on 8/5/16.
+//  Copyright © 2016 Lickability. All rights reserved.
+//
+
+import XCTest
+@testable import PinpointKit
+
+class SystemLogCollectorTests: XCTestCase {
+    
+    func testLogCollectorCollectsLogs() {
+        let testString = "TestLog"
+        let sut = SystemLogCollector(loggingType: .Testing)
```

Comment:
I know that `sut` is pretty standard terminology for tests, but it's can be kind of difficult to read / understand at first. Could we call it something like `logCollector` or some similarly descriptive name?